src/client/AppRouter.tsx

```diff
@@ -22,8 +23,9 @@ export const AppRouter: React.FunctionComponent<{}> = () => {
                     component={ClassroomPageContainer}
                 />
                 <Route path="/room/:roomId" component={PrivateRoomContainer} />
+                <Route path="/calendar-test" component={Calendar} />
                 <Route path="/login" component={LoginContainer} />
-                <Route path="/register" component={RegisterContainer} />
+                <Route path="/register" component={RegisterContainer} />]{" "}
```

Comment:
Is that bracket intended?

Reply:
I would remove these before merging.

Reply:
Yeh I just removed them. Sorry idk why they're there. I don't recall doing anything to that line.